tools/cbmc/proofs/ARP/ARP_FreeRTOS_OutputARPRequest/OutputARPRequest_harness.c

```diff
@@ -24,7 +24,17 @@ NetworkBufferDescriptor_t xNetworkBuffer;
  * of an ARPPacket to gurantee memory safety.
  */
 NetworkBufferDescriptor_t *pxGetNetworkBufferWithDescriptor( size_t xRequestedSizeBytes, TickType_t xBlockTimeTicks ){
-	xNetworkBuffer.pucEthernetBuffer = malloc(xRequestedSizeBytes);
+	#ifdef CBMC_PROOF_ASSUMPTION_HOLDS
```

Comment:
This is a stub that overrides the original code? If so, having a clean way of marking it as a stub would be nice.